docs/getting_started/installation.rst

```diff
@@ -169,21 +167,14 @@ Then it can be used during installation either like
 
 .. code-block:: bash
 
-  conda install -c conda-forge modin-ray modin-hdk --experimental-solver=libmamba
+  conda install -c conda-forge modin-ray modin- --experimental-solver=libmamba
 
 or starting from conda 22.11 and libmamba solver 22.12 versions
 
 .. code-block:: bash
 
-  conda install -c conda-forge modin-ray modin-hdk --solver=libmamba
-
-
-Using Intel\ |reg| Distribution of Modin
-""""""""""""""""""""""""""""""""""""""""
+  conda install -c conda-forge modin-ray --solver=libmamba
 
-With ``conda`` it is also possible to install `Intel Distribution of Modin`_, a special version of Modin
-that is part of Intel\ |reg| oneAPI AI Analytics Toolkit. This version of Modin is powered by :doc:`HDK</development/using_hdk>`
-engine that contains a bunch of optimizations for Intel hardware. More details to get started can be found in the `Intel Distribution of Modin Getting Started`_ guide.
```

Comment:
Are you planning to remove mention of `Intel Distribution of Modin` completely?

Reply:
I don't see a need to mention it as Intel Distribution of Modin mostly relied on HDK engine, as well as we were supporting `modin` package on conda intel channel. Since HDK is no longer supported and we are stopping support for modin package on conda intel channel, there is no need to refer to Intel Distribution of Modin as some special/different package. Instead, we will refer from Intel Distribution of Modin docs to the official Modin documentation, which is public.